onnxruntime/core/providers/cuda/math/softmax.h

```diff
@@ -20,7 +20,7 @@ struct AccumulateType<double> { using type = double; };
 template <typename T>
 using AccType = typename AccumulateType<T>::type;
 
-template <typename T>
+template <typename T, bool is_log_softmax>
 Status SoftMaxComputeHelper(
```

Comment:
Can we add support for LogSoftmax on CUDA with this change?

Reply:
Yes. This definitely doable. Do we have a customer model that needs this? If not, I will add it to our backlog.

Reply:
will be backlogged, close it for now

---
In reply to: [401274909](https://github.com/microsoft/onnxruntime/pull/3174#discussion_r401274909) [](ancestors = 401274909)